browser.json

```diff
@@ -125,12 +125,16 @@
       "to": "./menu-button[ds-4].js",
       "if-flag": "ds-4"
     },
-
     {
       "from": "./notice.js",
       "to": "./notice[ds-4].js",
       "if-flag": "ds-4"
     },
+    {
```

Comment:
I think we need an entry for each of the different types of notices.  Should have one for section and window as well.

Reply:
Good spot! Will fix. I notice I missed index.js and index.browser.json too

Reply:
Fixed.